conconartist/lyric-lava
# Feature/setup

#### Purpose: 
> Check all that apply.
### What does this PR do? 
> Clearly and concisely describe changes made. 

Basic setup with components and CSS 

### Problems Encountered & Resulting Solutions  
> State any problems encountered 
> Describe your code changes in detail for reviewers.


### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?

App.js, index.js were updated.  Form, Prompt, Load, and Error folders were all added.  CSS files were updated.

### How should this be tested?
> What steps need to be taken to confirm functionality? 

Run `npm start` in terminal and view the initial layout in browser.

### Steps To Take
> Any questions and possible steps moving forward.

Connecting to API and testing 

#### Checklist:
